types/aws-cloudfront-function/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for AWS CloudFront Functions 1.0
```

Comment:
```suggestion
// Type definitions for non-npm package AWS CloudFront Functions 1.0
```

This should avoid the need to turn off npm-naming

Reply:
Ok pushed and removed that rules from tslint.json